src/control-plane/billing/billing-provider.ts

```diff
@@ -54,6 +54,7 @@ export class BillingProvider extends Construct {
    */
   constructor(scope: Construct, id: string, props: BillingProviderProps) {
     super(scope, id);
+    addTemplateTag(this, 'BillingProvider');
```

Comment:
would have having the actual billing provider that the customer is using be more useful?

We could have implementers of IBilling specify a name (similar to `idpName`) and we just surface that here?